Tasks/Kudeiko/Task6/Task6_5.py

```diff
@@ -0,0 +1,35 @@
+# 5. Write a recursive function to find the Nth number in the Fibonacci sequence.
+
+class Error(Exception):
+    pass
+
+
+class NumberLessThanOneError(Error):
+    """"Вызывается когда значение менее 1"""
+    pass
```

Comment:
Why do you need this intermediate Error class? It makes no sense. But still the inheritance is cool)

Reply:
On the site where I read about exceptions, they suggested putting all copyright exceptions into one class. I'm just starting to understand classes.